app.js

```diff
@@ -4,7 +4,6 @@ const cardContainer = document.getElementById('books');
 
 class Book{constructor(title, author){ this.title = title; this.author = author }}
 
- 
```

Comment:
https://github.com/YannickZahinda/Awesome-book/blob/69dd6c82521046bdb291a274c0754740f0250498/app.js#L50-L52

- [ ] Do not commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website. Kindly remove this.